packages/zwave-js/src/lib/commandclass/SceneActuatorConfigurationCC.ts

```diff
@@ -0,0 +1,417 @@
+			if (typeof value !== "number") {
+				throwWrongValueType(
+					this.ccId,
+					property,
+					"number",
+					typeof value,
+				);
+			}
+
+			// We need to set the dimming duration along with the level
+			const node = this.endpoint.getNodeUnsafe()!;
+			// If duration is missing, we set a default of instant
+			const dimmingDuration =
+				node.getValue<Duration>(
+					getDimmingDurationValueID(this.endpoint.index, propertyKey),
+				) ?? new Duration(0, "seconds");
+			await this.set(propertyKey, dimmingDuration);
+		} else {
+			// setting dimmingDuration value alone not supported,
+			// to be added when setting Duration values is supported
+			throwUnsupportedProperty(this.ccId, property);
+		}
+
+		// Verify the current value after a delay
+		this.schedulePoll({ property, propertyKey });
+	};
+
+	protected [POLL_VALUE]: PollValueImplementation = async ({
+		property,
+		propertyKey,
+	}): Promise<unknown> => {
+		switch (property) {
+			case "level":
+			case "dimmingDuration": {
+				if (propertyKey == undefined) {
+					throwMissingPropertyKey(this.ccId, property);
+				} else if (typeof propertyKey !== "number") {
+					throwUnsupportedPropertyKey(
+						this.ccId,
+						property,
+						propertyKey,
+					);
+				}
+				return (await this.get(propertyKey))?.[property];
+			}
+			default:
+				throwUnsupportedProperty(this.ccId, property);
+		}
+	};
+
+	public async set(
+		sceneId: number,
+		dimmingDuration: Duration,
+		level?: number,
+	): Promise<void> {
+		this.assertSupportsCommand(
+			SceneActuatorConfigurationCommand,
+			SceneActuatorConfigurationCommand.Set,
+		);
+
+		const cc = new SceneActuatorConfigurationCCSet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+			sceneId,
+			dimmingDuration,
+			level,
+		});
+
+		await this.driver.sendCommand(cc, this.commandOptions);
+	}
+
+	public async getActive(): Promise<
+		| Pick<
+				SceneActuatorConfigurationCCReport,
+				"sceneId" | "level" | "dimmingDuration"
+		  >
+		| undefined
+	> {
+		this.assertSupportsCommand(
+			SceneActuatorConfigurationCommand,
+			SceneActuatorConfigurationCommand.Get,
+		);
+
+		const cc = new SceneActuatorConfigurationCCGet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+			sceneId: 0,
+		});
+		const response = await this.driver.sendCommand<SceneActuatorConfigurationCCReport>(
+			cc,
+			this.commandOptions,
+		);
+
+		if (response) {
+			return pick(response, ["sceneId", "level", "dimmingDuration"]);
+		}
+	}
+
+	public async get(
+		sceneId: number,
+	): Promise<
+		| Pick<SceneActuatorConfigurationCCReport, "level" | "dimmingDuration">
+		| undefined
+	> {
+		this.assertSupportsCommand(
+			SceneActuatorConfigurationCommand,
+			SceneActuatorConfigurationCommand.Get,
+		);
+
+		if (sceneId === 0) {
+			throw new ZWaveError(
+				`sceneId 0 is not valid for get. Use getActive() instead.`,
+				ZWaveErrorCodes.Argument_Invalid,
+			);
```

Comment:
I'm not sure about the wording of this error.  It feels a bit cryptic, but I didn't want to make it too long.

Reply:
Maybe `Invalid scene ID 0. To get the currently active scene, use getActive() instead.`